primitive-types/Cargo.toml

```diff
@@ -15,7 +15,8 @@ impl-serde = { version = "0.3.1", path = "impls/serde", default-features = false
 impl-codec = { version = "0.6.0", path = "impls/codec", default-features = false, optional = true }
 impl-num-traits = { version = "0.1.0", path = "impls/num-traits", default-features = false, optional = true }
 impl-rlp = { version = "0.3", path = "impls/rlp", default-features = false, optional = true }
-scale-info-crate = { package = "scale-info", version = ">=0.9, <2", features = ["derive"], default-features = false, optional = true }
+# See https://github.com/paritytech/parity-common/pull/556#issuecomment-872127496 why this is here:
+scale-info-crate = { package = "scale-info", version = "2", features = ["derive"], default-features = false, optional = true }
```

Comment:
@wigy-opensource-developer shouldn't this be `version = "3"`?

Reply:
parity-scale-codec is now v3, and scale-info v2 depends on it. Funsies.

Reply:
I think you're conflating parity-scale-codec (v3) with scale-info (v2)